src/hist.jl

```diff
@@ -115,6 +115,37 @@ end
 abstract type AbstractHistogram{T<:Real,N,E} end
 
 # N-dimensional histogram object
+"""
+    Histogram <: AbstractHistogram
+
+The `Histogram` type represents data that has been tabulated into intervals
+(known as *bins*) along the real line, or in higher dimensions, over a real space.
```

Comment:
I've changed "plane" to "space" since AFAIK the former refers to a 2D space only. Right? (This was in the original docs.)

Reply:
Sure!